cylc/flow/task_job_mgr.py

```diff
@@ -190,14 +190,11 @@ def poll_task_jobs(self, suite, itasks, poll_succ=True, msg=None):
         pollable_statuses = {
             TASK_STATUS_SUBMITTED, TASK_STATUS_RUNNING, TASK_STATUS_FAILED
         }
-        if poll_succ:
-            pollable_statuses.add(TASK_STATUS_SUCCEEDED)
         for itask in itasks:
             if itask.state(*pollable_statuses):
                 to_poll_tasks.append(itask)
```

Comment:
Do we even need this logic any more?

Shouldn't all task in the regular pool be pollable?

Reply:
Agreed